modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Details.tsx

```diff
@@ -71,8 +73,6 @@ const Details = ({
 					fdsViewLabelRef.current?.value ?? ''
 				);
 			}
-
-			onFDSViewUpdate(responseJSON);
```

Comment:
@rubenvillarnet This caused SF error in CI. Why are we removing this?

Reply:
@markocikos SF passed!